data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -136,6 +136,17 @@ local raid(level='raid0', uefi=false) = {
         mdswap_partition,
       ],
     },
+  ] else if boot_type == 'prep' then [
+    // PowerPC PReP boot partition case
```

Comment:
it is well known that for other architectures there are special boot partitions, you can omit the comment.